commands/reveal.js

```diff
@@ -1,8 +1,23 @@
+const Game = require('../src/game.js');
+
 module.exports = {
-	name: 'reveal-roles',
-	description: 'Reveals the roles from the previous game',
-	execute(client, message, args) {
-		console.log(message);
-        console.log(args);
-	},
-};
\ No newline at end of file
+  name: 'reveal',
+  description: 'Reveals the roles from the previous game',
+  execute(client, message, args) {
+    if (message === '') {
+      return message.channel.send('No Sheriff Game name given.');
+    }
```

Comment:
Do we need lines 7-9 since we have lines 13-15?
What happens when a user doesn't have a game in their name?

Reply:
You're right, now if no game name is given then it will return the most recent game created for that user. So yes, this is not needed.

Reply:
I guess if there isn't a game for the user the catch will report that.